tools/system_libs.py

```diff
@@ -1431,6 +1431,21 @@ def can_use(self):
     return settings.WASMFS
 
 
+# Libc components of WasmFS, which uses more libc code than the JS filesystem.
+# This must be a separate library than libwasmfs because the build commands
+# are different for libc code.
```

Comment:
I hope this isn't necessary.   

There are a couple of ways we can get the correct build flags:

1. Have libwasmfs inherit from MuslInternalLibrary.
2. Use the new `customize_build_cmd` to use different cflags for utime.c.

Reply:
Ok, switched to a single library and picking the build flags in `customize_build_cmd`.